### ISSUE TITLE
Background color not working

### ISSUE THREAD

**Issue by touilleMan** [lektor/lektor-markdown-highlighter#2]
It seems lektor's generated html for a code block has it class name set to `highlight`:

``` html
<div class="highlight">
  <pre>
    <span></span>
    <span class="o">&gt;&gt;&gt;</span>
    <span class="kn">import</span> <span class="nn">os</span>
  </pre>
</div>
```

However in pygments the correct class is `hll` (for example in monokai: https://github.com/richleland/pygments-css/blob/master/monokai.css#L1)

I'm not sure the bug is in this extension or lektor itself though.

**Comment by kiddten** [comment]
Same here :(

**Comment by nixjdm** [comment]
fixed in pr #4

**Comment by nixjdm** [comment]
I just made this a new release by the way.
